src/styles/index.scss

```
@@ -1,3 +1,157 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  padding: 0;
+  font-family: "Roboto", Arial, sans-serif;
+  --white: #fff;
+  --color-secondary: #616070;
+  --color-main: #060b35;
+  --background-color: #00acdc;
+  background-color: #dfdfdf;
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: var(--white);
+
+  &__image {
+    margin-bottom: 36px;
+  }
+}
+
+.image {
+  padding: 32px 19px 0;
+
+  &__product {
+    width: 160px;
+    height: 134px;
+  }
+}
+
+.card__info {
```

```suggestion
&__info {
```
Consider using scss nesting